### PR TITLE
Process localizer data & fix bug in ICA (didn't operate on existing filtered data)

### PR DESCRIPTION
**Pull request by hoechenberger**
cc @agramfort

### CODE REVIEW

**Comment by hoechenberger** [comment]
Currently only sensor-level analysis of a single participant

**Comment by hoechenberger** [comment]
@agramfort I'm afraid the test fails due to high memory consumption, any idea?

**Comment by jasmainak** [comment]
ICA is usually the culprit :) do you decimate?

**Comment by hoechenberger** [comment]
@jasmainak Yep, I decimate to 250 Hz, but maybe the recording is just too long. Profiling to see what I can do.

**Comment by hoechenberger** [comment]
No wait… this is resampling, not decimation. Mh!

**Comment by jasmainak** [comment]
in the ICA constructor you can give a `decim` parameter and you can also give a `tmax` I believe

**Comment by hoechenberger** [comment]
Ok it's working now. Turns out we **didn't** use the filtered & resampled data in ICA!! So the CI limitation helped us find a bug.

We now operate on the filtered & resampled data. One thing we'll need to discuss is whether it's actually okay to apply a second high-pass filter run on this data, which will currently happen if `ica_l_freq > l_freq`. Will this cause trouble?

Otherwise, we could start with the Maxwell-filtered data (without a frequency filter applied) and do everything anew, cleanly. But this would potentially waste processing time.

File: scripts/preprocessing/04a-run_ica.py [hunk 1]
```diff
@@ -59,7 +68,11 @@ def filter_for_ica(raw, subject, session):
     """Apply a high-pass filter if needed."""
     if config.ica_l_freq == config.l_freq or config.ica_l_freq is None:
         # Nothing to do here!
-        msg = 'Not applying high-pass filter.'
+        msg = 'Not applying high-pass filter '
+        if config.ica_l_freq == config.l_freq:
```
Comment (agramfort) [inline scripts/preprocessing/04a-run_ica.py:72]:
```suggestion
        if config.ica_l_freq <= config.l_freq:
```

Comment (agramfort) [reply scripts/preprocessing/04a-run_ica.py:72]: you should high pass filtered higher if needed

Comment (hoechenberger) [reply scripts/preprocessing/04a-run_ica.py]: Good catch, I have addressed this in 63dc25a

**Pull request merged**